demo/lemonldap/lmConf-1.json

```diff
@@ -374,6 +374,7 @@
   "oidcRPMetaDataOptions": {
     "james": {
       "oidcRPMetaDataOptionsAccessTokenClaims": 1,
+      "oidcRPMetaDataOptionsAccessTokenExpiration": 120,
```

Comment:
120... seconds? Isn't that rather short? Or is it an other duration measure?

Reply:
Yes short, but this is to test refresh_token (see my comment)